test/DynamicEater.test.ts

```
@@ -98,10 +98,28 @@ describe("Dynamic Merge / Eater Contract", function () {
 			expect(totalVested).to.be.greaterThan(0);
 		});
 
-		it("Should add user to white list", async () => {
+		it.only("Should be able to merge when merkle root is bytes 0", async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)